packages/regl-worldview/src/Worldview.js

```diff
@@ -38,6 +38,10 @@ export type BaseProps = {|
   backgroundColor?: Vec4,
   // rendering the hitmap on mouse move is expensive, so disable it by default
   hitmapOnMouseMove?: boolean,
+  // we avoid rendering the hitmap on mouse down by default
+  hitmapOnMouseDown?: boolean,
+  // we avoid rendering the hitmap after dragging by default
+  hitmapAfterMouseDrag?: boolean,
```

Comment:
can we instead have a `disableHitmapForEvents: MouseEventEnum[]` prop, and default it to `["mouseMove"]`, but overridable using `hitmapOnMouseMove`? And then treat `hitmapOnMouseMove` as legacy and log a warning when it is used, and mark it as deprecated in the docs? We'll also have to add these props to the docs.

As currently written this will change the default hitmap behavior for anyone upgrading the library, so I'm a bit leery of adding it like this. @troygibb what do you think?

Reply:
Yes, I think that would be great. I also prefer not to change the current behavior.

Reply:
@davidswinegar Check out my latest changes. I added the `disableHitmapForEvents` prop, as well as new stories and docs. 
If `hitmapOnMouseMove` is provided, it will override `disableHitmapForEvents` on mouse move.

Reply:
Hm, this new model doesn't seem to do `hitmapAfterMouseDrag` any more, or does it? We still do render a hitmap on mouse up after a drag now, even though for Webviz we don't really need to any more?

Reply:
That's right. I just updated the PR description.

`hitmapAfterMouseDrag` is no longer needed, so I removed it in my latest commit. My initial goal was to render a hitmap only if the user has clicked (that is, mouseDown then mouseUp without moving), which is exactly what the `onClick` event does.

Still, if we need to support rendering after a mouse drag, we just do:
```
disableHitmapForEvents = ["onMouseDown", "onMouseMove"];
```

Does that make sense?

Reply:
Ah cool, I didn't realise we already have `onClick`; that makes a ton of sense then. 👍